packages/suite/src/views/wallet/coinmarket/buy/offers/Offers/List/index.tsx

```diff
@@ -47,9 +45,10 @@ const OrigAmount = styled.div`
 
 const StyledIcon = styled(Icon)`
     padding: 0 10px;
+    margin: 0 20px;
 `;
 
-const Text = styled.div`
+const Text = styled(H2)`
```

Comment:
just a nitpick, check this https://styled-components.com/docs/basics#attaching-additional-props
```suggestion
const Text = styled(H2).attrs({
  fontWeight: 400,
})`
```
then you could get rid off all those `fontWeight={400}` on every use...